metadata.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.1",
   "author": "puppetlabs",
   "summary": "Puppet module providing the certregen face for regenerating CA certificates",
-  "license": "Puppet Enterprise Software License Agreement",
+  "license": "proprietary",
```

Comment:
This conflicts with the contents of the `NOTICE` file that indicates that the module is Apache licensed.